source/installguide/management-server/_database.rst

```diff
@@ -197,6 +197,17 @@ MySQL. See :ref:`install-database-on-separate-node`.
       cluster management server node IP. If not specified, the local IP
       address will be used.
 
```

Comment:
option description should be fine here, any generic statements can be moved to the script description.

Reply:
Done